### ISSUE TITLE
Chart JS dependency error

### ISSUE THREAD

**Issue by noorahmed94** [bedezign/yii2-audit#165]
I am facing composer error while installing this plugin. I think its an issue in 2amigos/yii2-chartjs-widget as it needs bower-asset/chartjs but chartjs has been moved to npm-asset/chartjs. 

```
Problem 1
    - 2amigos/yii2-chartjs-widget 2.0.1 requires bower-asset/chartjs 2.1.6 -> no matching package found.
    - bedezign/yii2-audit 1.0.7 requires 2amigos/yii2-chartjs-widget 2.0.1 || 3.* -> satisfiable by 2amigos/yii2-chartjs-widget[2.0.1].
    - Installation request for bedezign/yii2-audit 1.0.7 -> satisfiable by bedezign/yii2-audit[1.0.7].

```

Any help?

**Comment by noorahmed94** [comment]
Any update??

**Comment by cornernote** [comment]
I believe this was fixed with #159

**Comment by noorahmed94** [comment]
resolved.
